cmd/registry/core/print.go

```diff
@@ -152,7 +159,8 @@ func PrintArtifactContents(artifact *rpc.Artifact) error {
 	case "gnostic.openapiv3.Document":
 		return unmarshalAndPrint(artifact.GetContents(), &openapiv3.Document{})
 	default:
-		fmt.Printf("%+v", artifact.GetContents())
+		// To get a parent context here would require changing the ArtifactHandler type in handlers.go
+		log.FromContext(context.TODO()).Fatalf("Unsupported message type: %s, please use --raw to get raw contents", messageType)
```

Comment:
Can this be returned as an error that gets logged by the caller?

Reply:
Thanks, I missed that - I'm returning an error now and changed the caller to log it at Error instead of Debug.